server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/redirect/NotifyMailetsMessage.java

```diff
@@ -90,6 +95,15 @@ public String generateMessage(String parameterMessage, Mail originalMail) throws
         return builder.toString();
     }
 
+    @VisibleForTesting static Optional<Long> getMessageSizeEstimation(Mail mail) {
+        try  {
+            return Optional.of(mail.getMessageSize())
+                .filter(size -> size > 0);
+        } catch (MessagingException e) {
+            return Optional.empty();
```

Comment:
don't we need to log this failure? at least in debug?

Reply:
done